### PR TITLE
starting work to display RelatedArtists on Artwork page

### PR DESCRIPTION
**Pull request by oxaudo**
Thanks @mzikherman for walking me through graphQl for this. 

Adding the code and display to put some similar artists on the artwork page as requested by: https://artsyproduct.atlassian.net/browse/DISCO-565

Current look:
screen shot 2018-12-17 at 4 50 58 pm

This is missing some specs and also probably needs some fiddling with responsive behavior here, but it does seem to work:)

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d2741bbce36b6a6b7300f93744377bc2ac7a82a7

https://deploy-preview-1713--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkRelatedArtists.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { ArtworkRelatedArtists_artwork } from "__generated__/ArtworkRelatedArtists_artwork.graphql"
+
+export interface ArtworkRelatedArtistsProps {
+  artwork: ArtworkRelatedArtists_artwork
+}
+
+export class ArtworkRelatedArtists extends React.Component<
+  ArtworkRelatedArtistsProps
+> {
+  render() {
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtworkRelatedArtists.tsx]: This component should probably check the length of the related edges, and cleanly not render the rail if there aren't any related results (there might not be for reasons).

**Comment by mzikherman** [comment]
Cool, yea nice and simple! I'll merge so we can get it in and iterate.

File: src/Apps/Artwork/Components/ArtworkRelatedArtists.tsx (hunk 1 above)
Comment (oxaudo) [reply src/Apps/Artwork/Components/ArtworkRelatedArtists.tsx]: 👍 Will add!

**Pull request merged**